data/hai/hai reveal 1 intro.txt

```diff
@@ -11,47 +11,461 @@
+	to offer
+		has "event: hr: pirates respond to leaks"
+		has "First Contact: Unfettered: offered"
+		not "Hai Reveal Concern Period: Raid on Allhome (local): offered"
+		random < 20
+	on offer
+		set "hr: allhome raided"
+		conversation
+			`A wall screen situated between two food stands just outside the docking area is playing through a news segment, and your ears suddenly perk up at the word "marauder".`
+			`	"Yesterday, a marauder terrorized the inhabitants of Allhome. They arrived by posing as a trader to investigate rumors currently circulating in human space about our existence. Hai security failed to identify the danger, and the individual snuck his ship in and landed at the spaceport without any opposition. Then his crew hijacked a vessel with some of the crew still on board and made off with it successfully. At this time it is unclear what has happened to those crewhai."`
+			`	At a table nearby, a Hai says, "Oh, that's awful!" and the human next to her responds, "Feels like being back on Solace."`
+			choice
+				`	(Approach the pair.)`
+				`	(Keep walking.)`
+					goto walking
+			
+			`	As you approach the human, a relatively young-looking man, glances up at you. "Oh hello, just arrived did you? You didn't happen to see an Argosy out there on your way in?"`
+			`	"Oh shush," says the Hai, "Your uncle will be here when he said he will, no need to bother random folks just going about their business."`
+			`	She looks at you with a wary eye, but then seems to relent. "Though I suppose you did come over here of your own volition didn't you. Can we help you at all, Captain?"`
+			choice
+				`	"Is this a common problem for Allhome?"`
+				`	"I heard you say this happened on Solace?"`
+					goto solace
+					
+			`	"Oh imperishables, no, not at all. It's completely unprecedented for something like that to happen, and in broad daylight too from the looks of the footage," she says.`
+			`	"It's like I was saying before, Auntie, ever since the retraction and that advisory, I swear no one except pirates is trying to get through the wormhole anymore."`
+			`	He looks at you seriously. "It's getting so bad that it's starting to feel like it did during the war, when all the militia were away and before the Free Worlds got a handle on the pirates. It seemed like every new ship that arrived was there to try and rob us, and it's the same now. We came here to be safe, but now it feels like all the government has managed to do is make sure that no one friendly is willing to risk the journey..."`
+				goto resolve
+			
+			label solace
+			`	The young man sits up a little straighter. "When the war started and most of the militia went away to the front, before the Free Worlds got a handle on the pirates, Solace essentially came under siege. Other than the regular merchants that we were familiar with, it seemed like every new ship that arrived was there to try and rob us. Either raiding the regulars, or raiding the planet.`
```

Comment:
Hey Grey, I have some notes somewhere about the founding of the Southern Defense Pact (the explanation of its early history was going to be needed for a mission), and a major factor of its founding was that Solace was put under siege by pirates for some time – the lack of suitable Navy garrison on a planet as important as Solace was what made the Southern Rim decide to put their own defense into their own hands. I placed this in the late 2990s or early 3000s, I think. So if I can find those notes and share some more details, could this dialogue be changed to reference that?

Reply:
I don't think this particular scene works if the event wasn't during the actual war? Though we can explore the timeline a little more fully.
I think that's generally a good lore bit to have though, just need to look into how/if/where to weave it in a little more thoroughly.

Reply:
I imagine that if Solace was really under siege like the young man says the Free Worlds' support in that region would probably collapse. Maybe he's exaggerating, but Solace isn't any old planet, it's a major manufacturer of engines for the Rim's shipyards. If the Free Worlds lost that, the Free Worlds would probably struggle a lot more against the Navy.

Reply:
He *doesn't* say it was under siege, he says it was *effectively* under siege.
It's not like there was actively a pirate fleet there keeping people out - it's that there were enough pirates moving around unobstructed generally that _new_ people simply did not come to visit.

Solace is a major destination world. *New* people would be the lifeblood of the businesses around the spaceport and various industries beyond.
As such, the planet could have its core supply chains be only minimally affected by pirate activity - but it would still feel like the world had been cut off.

Reply:
This would be especially true if they had had an actual siege in recent memory - because to anyone not directly in the core logistics/supply chain industries, it would look identical.

Reply:
Unfortunately, none of this is the sort of stuff that a traumatised teen who's in the middle of despairing is going to be likely to say.

Reply:
That's a fair point, yea.